Gemfile

```diff
@@ -12,6 +12,10 @@ group :development, :test do
   gem 'rspec', '~> 3.2'
   gem 'rake-compiler'
   gem 'ruby-prof', :platforms => :mri
+  gem 'benchmark-ips'
+  if RUBY_VERSION >= "2.1"
+    gem 'memory_profiler', :platforms => :mri
+  end
```

Comment:
We don't need it for the content of the Gem - yeah that is correct - should I add this in a comment or something to the gist and remove it from here?